File: parse_pdf.py
import sys
try:
    import fitz  # PyMuPDF
except ImportError:
    print("PyMuPDF not installed")
    sys.exit(1)

doc = fitz.open('/cv.pdf')
text = ""
for page in doc:
    text += page.get_text()

with open('/app/cv_text.txt', 'w', encoding='utf-8') as f:
    f.write(text)

print("PDF parsed successfully")

File: parse_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('/cv.pdf');

pdf(dataBuffer).then(function(data) {
    fs.writeFileSync('/app/cv_text.txt', data.text);
    console.log('PDF parsed successfully');
}).catch(function(error) {
    console.error('Error parsing PDF:', error);
});
